packages/ppxfind/ppxfind.1.4/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Tool combining ocamlfind and ppx"
+description: """
+Ppxfind is a small command line tool that among other things allows
+to use old style ppx rewriters with jbuilder.
+"""
+maintainer: ["Jérémie Dimino <jeremie@dimino.org>"]
+authors: ["Jérémie Dimino <jeremie@dimino.org>"]
+license: "BSD3"
+homepage: "https://github.com/diml/ppxfind"
+doc: "https://diml.github.io/ppxfind/"
+bug-reports: "https://github.com/diml/ppxfind/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml-migrate-parsetree"
+  "ocamlfind"
+  "ocaml" {>= "4.02.3"}
+]
+conflicts: [
+  "dune" {= "1.2" | = "1.2.1"}
+]
```

Comment:
This conflict is redundant since dune >= 2.0 is already required.

Reply:
Indeed, I fixed the upstream repo, thanks!